### PR TITLE
guides: Add instructions for multiple SSG tools

### PR DESCRIPTION
**Pull request by davlgd**
## Checklist

- [ ] My PR is related to an opened issue : #

**⚠️ If changes affect GitHub Action files**

- [ ] I have added tests to cover my changes : [link](url)

## Pull request type

Please check the type of change your PR introduces:

- [x] Documentation content changes
- [ ] Bugfix on the site
- [ ] Build related changes
- [ ] Other (please describe):
      
## Description

New guides for multiple SSG:
- Astro
- Docusaurus
- 11ty
- Hexo
- Hugo
- Lume/Deno
- mdBook
- MkDocs
- Nuxt

I used partials for shared instructions and added SVG icons

## Reviewes
_Who should review these changes?_ @

### CODE REVIEW

**Review by zwindler — APPROVED** [review]

**Comment by juliamrch** [comment]
## Build folder

It should be noted that folders like `/dist` or `/build` in JavaScript/Typescript based frameworks aren't committed by default, and that Clever Cloud permission levels will log "permission denied" when trying to create it and use it as webroot. 

In these tutorials, we need to specify that the user must: 

1. Build locally the site before the first deploy
2. Commit the build folder

Files inside the `/dist` and `/build` can be omitted, because they will be recreated again while deploying.

**Review by juliamrch — CHANGES_REQUESTED** [review]
I find all these new guides amazing. Thanks for all this work.

Only change to ensure presentation consistency and readability is to replace the title by the framework name, and we should be good to go.

**Comment by davlgd** [comment]
I'm not sure so short titles will be good for SEO. Can you check with marketing team?

**Comment by juliamrch** [comment]
> I'm not sure so short titles will be good for SEO. Can you check with marketing team?

It's not a SEO but a readability issue. Readability and clarity in sidebar for the doc is always privileged, and yes, even over SEO. Check the rest of the titles in `/guides`:  it's the framework name, and that's it. The SEO is handled by the description in metadata.

**Comment by davlgd** [comment]
The metadata is not enough for a good SEO strategy, that's my point. We also need these guides to be found/read. I agree on the importance of readability, but it can't be only one or the other. It's why I asked to involve marketing team to reach a consensus on this.

File: content/guides/_index.md
Comment (juliamrch) [inline content/guides/_index.md:1]: `/guides` is the root of the page (framework-first logic and not runtime). `Static` root should be removed.

File: content/guides/static/_index.md
Comment (juliamrch) [inline content/guides/static/_index.md:1]: idem, this should be put in `/guides/_index.md`

File: static/partials/guides/cards-static.md
Comment (juliamrch) [inline static/partials/guides/cards-static.md:1]: replace current path by `/guides/static/<framework-name`

**Review by juliamrch — CHANGES_REQUESTED** [review]
Replace path to avoid hiding frameworks in another level and let them be indexed by alphabetical order as intended for this page.

**Comment by davlgd** [comment]
If it's ok for marketing team, it's ok for me

File: content/guides/_index.md
Comment (davlgd) [reply content/guides/_index.md:1]: If I do that, all static guides will be on first level, we can't stay on such a flat structure on the long term (more guides > more menu entries, too long menu). And we won't have a static guides). I can do that now but we'll have to change later (and thus change URLs).

Comment (juliamrch) [reply content/guides/_index.md:1]: There isn't runtime-logic in this page, but framework-first logic, automatically indexed by alphabetical order so people can fin immediately their framework in the side-bar. However, we can keep the cards as they are, to include them in a link like this : http://clever-doc-review-davlgd-static-guides.cleverapps.io/guides/#static-site-generator-ssg

File: content/guides/_index.md [hunk 1]
```diff
@@ -36,6 +36,9 @@ comments: false
   
 {{< /cards >}}
 
+## Static Site Generator (SSG)
+{{< readfile file="guides/cards-static.md" >}}
```
Comment (juliamrch) [inline content/guides/_index.md:40]: Simply putting the content of `/cards-static.md` should be enough, do we need a partial for it?.

**Review by juliamrch — APPROVED** [review]
Subjects left: 

- Do we need a partial for the cards, since we a a single level in `/guides/_index/md`?
- Subtitle partial missing to keep the look consistent

I can do these changes after merging and before pushing to prod, it's not a big issue

**Review by juliamrch — DISMISSED** [review]

**Review by juliamrch — APPROVED** [review]

**Comment by juliamrch** [comment]
There were some conflicts in icons.yml` that I just fixed (by setting current fluent and grade icons). Could you review it @davlgd ? And if everything is ok we can merge the new guides.

**Comment by davlgd** [comment]
It's ok for me

**Pull request merged**